### DISCUSSION TITLE
Why am I getting an error spawning 'typescript-language-server' despite being installed?

### DISCUSSION THREAD

**Discussion by zeitchef** [williamboman/nvim-lsp-installer#803] [Q&A] [answered]
Not sure how long this has been happening, but for a few days I'm getting this error every time I open an TypeScript file:

<img width="1251" alt="Screen Shot 2022-06-16 at 10 14 02" src="https://user-images.githubusercontent.com/6670659/174025801-073915a0-5102-4af4-b671-d98b56a6fccc.png">

Output of `:LspInfo` shows that the server is attached:

<img width="1440" alt="Screen Shot 2022-06-16 at 10 14 49" src="https://user-images.githubusercontent.com/6670659/174026022-b1f9f3b9-5078-478e-9f2e-0baaae428a53.png">

And `:LspPrintInstalled` proves it is indeed installed. Any idea why I might be getting this error then?

**Comment by williamboman** [comment, marked as answer]
Hello! Do you set up nvim-lsp-installer before you set up any servers? `require("nvim-lsp-installer").setup {}`

**Reply by zeitchef** [reply]
Hey @williamboman - you know what, after I posted I checked the docs again and realized that now everything is configured really differently! So, I spent an hour last night refactoring my entire LSP setup. So far, so good and a LOT simpler to setup. Thanks again, and sorry for the noise!
